metanode/partition_store_ticket.go

```diff
@@ -162,3 +166,14 @@ func (mp *metaPartition) stop() {
 		close(mp.stopC)
 	}
 }
+
+func resetTimer(timer *time.Timer, ts int64) {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	timer.Reset(time.Duration(ts+r.Int63n(RandomIntervalS)) * time.Second)
+}
+
+// Avoid devices starting scheduled tasks simultaneously
+func resetTimerWithRandomTime(timer *time.Timer, maxts int64) {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	timer.Reset(time.Duration(float32(maxts)*r.Float32()) * time.Second)
```

Comment:
here is max*[0,1), it's better to update as max*[1, 2), to avoid snapshot too frequently, you can you max+r.rand(max)

Reply:
and please to upate your graph after change.

Reply:
Fixed. And the graph will coming soon.